src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -27,23 +27,32 @@
 					name="termsAgreement"
```

Comment:
Only auto formatting changes for this file. Not sure why the formatting was so whacky.